src/main/java/com/synopsys/integration/detect/configuration/DetectableOptionFactory.java

```diff
@@ -224,8 +218,8 @@ public PipenvDetectableOptions createPipenvDetectableOptions() {
     public PipInspectorDetectableOptions createPipInspectorDetectableOptions() {
         String pipProjectName = getNullableValue(DetectProperties.DETECT_PIP_PROJECT_NAME);
         List<Path> requirementsFilePath = getValue(DetectProperties.DETECT_PIP_REQUIREMENTS_PATH).stream()
-            .map(it -> it.resolvePath(pathResolver))
-            .collect(Collectors.toList());
+                                              .map(it -> it.resolvePath(pathResolver))
```

Comment:
I'm pretty sure I was last in this file. I recently verified my formatter is up to date. Can you double check yours?